plugin/core/types.py

```diff
@@ -605,24 +607,34 @@ def map_from_remote_to_local(self, uri: str) -> Tuple[str, bool]:
         return _translate_path(uri, self._remote, self._local)
 
 
+NodeIpc = collections.namedtuple('NodeIpc', 'parent_conn,child_conn')
```

Comment:
How do I type this? I tried `NodeIpc.__annotations__`, but pyright still wouldn't pick it up. And looks like I can't use `typing.NamedTuple`, since it's a 3.5+ exclusive.

Reply:
I changed this to a class:

https://github.com/sublimelsp/LSP/pull/2015/files#diff-476d0275847cfb60ebc00c73c76d5c5e6595f5b70dd8c659d4f0fd1a854f49eeR610